api/KeyboardLayoutMap.json

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
KeyboardLayoutMap is defined here: https://wicg.github.io/keyboard-map/#keyboardlayoutmap-interface and it has "maplike" members (https://heycam.github.io/webidl/#idl-maplike).
So, you've added `entries`, `keys`, `size`, `values`. I think we also need `forEach`, `get`, and `has` then.

Reply:
Not sure how I missed get and has. It doesn't look like forEach is part of maplike: https://heycam.github.io/webidl/#es-maplike

Reply:
I would expect maplikes to have that method and if I inspect the object in Chrome Canary, there is `forEach`:

![keybboardlayoutmap](https://user-images.githubusercontent.com/349114/45417015-78e35180-b681-11e8-8983-1d92f021aecc.png)